.github/workflows/ci.yml

```
@@ -75,6 +75,7 @@ env:
   DO_RELEASE: ${{ github.event_name == 'release' || startsWith(github.event.inputs.do-release, 'v') }}
   DO_MORE_TESTS: ${{ github.event_name == 'schedule' || github.event.inputs.test-more == 'true' }}
   ESPTOOL_VERSION: "v4.8.1%2Btoitlang"
+  TEST_PI_HW: "true"
```

Somehow, I like grouping the boolean flags that decide on how much to run, build, test under the `DO_` prefix.

```suggestion
  DO_TEST_PI_HW: "true"
```

---

done.